java/src/Ice/src/main/java/com/zeroc/Ice/_ObjectPrxFactoryMethods.java

```diff
@@ -3,12 +3,14 @@
 package com.zeroc.Ice;
 
 /**
- * @hidden Provides overloads of the proxy factory methods with covariant return types so that
- *     applications do not need to downcast to the derived proxy type.
```

Comment:
Another place where the code was actually changed.

Looking back through the history, `@hidden` used to be on it's own line above the summary, and the formatter mucked it up. Tbf, the summary should come before any annotations anyways.